packages/emails/src/templates/OrganizationCreationEmail.tsx

```diff
@@ -0,0 +1,109 @@
+import { Trans } from "next-i18next";
+
+import { APP_NAME, WEBAPP_URL, IS_PRODUCTION } from "@calcom/lib/constants";
+
+import type { OrganizationCreation } from "../../templates/organization-creation-email";
+import { V2BaseEmailHtml } from "../components";
+
+export const OrganizationCreationEmail = (
+  props: OrganizationCreation & Partial<React.ComponentProps<typeof V2BaseEmailHtml>>
+) => {
+  const { prevLink, newLink, orgName: teamName } = props;
+  const prevLinkWithoutProtocol = props.prevLink?.replace(/https?:\/\//, "");
+  const newLinkWithoutProtocol = props.newLink?.replace(/https?:\/\//, "");
+  return (
+    <V2BaseEmailHtml subject={props.language(`email_organization_created|subject`)}>
+      <p style={{ fontSize: "24px", marginBottom: "16px", textAlign: "center" }}>
+        <>{props.language(`You have created ${props.orgName} organization.`)}</>
+      </p>
+      <img
+        style={{
+          borderRadius: "16px",
+          height: "270px",
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+        }}
+        src={
+          IS_PRODUCTION
+            ? `${WEBAPP_URL}/emails/calendar-email-hero.png`
+            : "http://localhost:3000/emails/calendar-email-hero.png"
```

Comment:
Doesnt WEBAPP_URL fall back to localhost - is the conditional needed?

Reply:
Yeah it should. Copied it from somewhere.  Nice catch !!

Fixed it now.